packages/adapter-vercel/index.js

```diff
@@ -238,12 +239,47 @@ const plugin = function (defaults = {}) {
 					src = '^/?';
 				}
 
-				src += '(?:/__data.json)?$';
-
 				const name = functions.get(pattern);
 				if (name) {
-					static_config.routes.push({ src, dest: `/${name}` });
-					functions.delete(pattern);
+					const isr = isr_config.get(route);
```

Comment:
If you don't have any route config, we'll never get into here because `functions` is empty (because of line 216)

Reply:
nice catch. fixed